contrib/hello-world/README.md

```diff
@@ -8,7 +8,7 @@ npm
 
 ##Configuration
 
-If you are not running a copy of Wasabi locally on your system at 192.168.99.100:8080, you will need
+If you are not running a copy of Wasabi locally on your system at localhost:8080, you will need
 to search for "WASABI.setOptions" in the source (currently two locations) and then change the server
```

Comment:
I would rather name the locations or leave out the "currently two locations" - Also maybe a hint for searching for localhost might be smarter: You don't mention Gruntfile.js or the shell scripts which one would need to change as well.